### ISSUE TITLE
Installed and unavailable

### ISSUE THREAD

**Issue by allmors** [binary-person/rammerhead#23]
I installed this project. But there is no response to access it, my firewall is all closed, all ports can pass

**Comment by darwapls** [comment]
I have the same issue. Can someone please guide on the proper way to setup https on the crossdomainport?

**Comment by binary-person** [comment]
my demo setup involves just proxying ssl with nginx. can you try that instead?

**Comment by allmors** [comment]
> my demo setup involves just proxying ssl with nginx. can you try that instead?

Yes, I configured ssl, 8433 prompts cross-domain, I can only use it with 443 port to use it normally.

![image](https://user-images.githubusercontent.com/93471589/175844365-a4b3a585-5e2e-472a-938d-bb53934ac8d0.png)

The cross-domain port must be consistent with the port used by the website to play the video normally, otherwise there will be a problem in the first picture of the feedback section, prompting cross-domain

![image](https://user-images.githubusercontent.com/93471589/175844543-6ef9dfd5-840f-4ddb-aa3d-e68f640b1542.png)


![image](https://user-images.githubusercontent.com/93471589/175845084-46e6e1d7-422f-4550-8b50-0765cb3668bd.png)

**Comment by binary-person** [comment]
hmm i see. yes, there are indeed some situations where simulating crossDomainPort will break things, bc of bad implementation.
keep it at 443 for now, while i find the time to fix this

**Comment by allmors** [comment]
> hmm i see. yes, there are indeed some situations where simulating crossDomainPort will break things, bc of bad implementation. keep it at 443 for now, while i find the time to fix this

thanks 😁
